php/commands/scaffold.php

```diff
@@ -446,6 +455,8 @@ function plugin( $args, $assoc_args ) {
 
 		if ( isset( $assoc_args['activate'] ) ) {
 			WP_CLI::run_command( array( 'plugin', 'activate', $plugin_slug ) );
+		} else if ( isset( $assoc_args['activate-network'] ) ) {
+			WP_CLI::run_command( array( 'plugin', 'activate', $plugin_slug), array( 'network' => true ) );
```

Comment:
That’s the same line you commented on earlier that you suggested I change to an `else if`, no? This line fixes #1709, while the other lines fix #1711.